### PR TITLE
Add api.Document.selectstart_event

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
One of the PRs that is intended to complete #1999 once and for all.  On https://developer.mozilla.org/en-US/docs/Web/API/Document/selectstart_event, it looks like the old compatibility table was removed before it was added to BCD. 
 This adds said data into the BCD.

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -9127,6 +9127,55 @@
           }
         }
       },
+      "selectstart_event": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Document/selectstart_event",
+          "description": "<code>selectstart</code> event",
+          "support": {
+            "chrome": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Document.json]: This should be `false`. [Here's the source](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/document.idl).

File: api/Document.json [hunk 2]
```diff
@@ -9127,6 +9127,55 @@
...
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Document.json]: Also `false`.

File: api/Document.json [hunk 3]
```diff
@@ -9127,6 +9127,55 @@
...
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Document.json]: `false`

**Comment by jpmedley** [comment]
I owe you an apology. Chromium should actually be true for many event handlers here marked false. Chrome implements these on a mixin. Here's [the inclusion in Document](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/document.idl?l=211) and here's [source for that inclusion](https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/dom/global_event_handlers.idl).

I'm guessing other browsers don't do it this way. I've stated elsewhere my belief that the goal of interface documentation should be to document as the developer sees it, not as it's represented in the uncompiled code or IDL. Document illustrates why the latter approach is not a good one.

**Comment by vinyldarkscratch** [comment]
Not a problem, it's a quick and easy fix!  Good catch, by the way.  :wink:  I think that you strike a good point -- web developers will see such events as a part of the Document API, as they won't see its parent.

**Comment by vinyldarkscratch** [comment]
Mirroring complete!

**Comment by chrisdavidmills** [comment]
@vinyldarkscratch argh, I think I've messed this up. I'm so sorry.

**Comment by vinyldarkscratch** [comment]
Haha, you're all good!  I was just about to push a merge of `master`, and I've still got the branch on my end.  Lemme send a follow-up PR and admin merge it.  :wink:

**Comment by chrisdavidmills** [comment]
@vinyldarkscratch phew ;-)

**Comment by vinyldarkscratch** [comment]
And done!

**Comment by chrisdavidmills** [comment]
Great stuff, thanks!

**Pull request merged**